lib/generators/test_unit/webauthn_authentication/templates/test/controllers/second_factor_authentications_controller_test.rb

```diff
@@ -0,0 +1,95 @@
+require "test_helper"
+require "webauthn/fake_client"
+
+class SecondFactorAuthenticationsControllerTest < ActionDispatch::IntegrationTest
+  setup do
+    @user = users(:one)
+    @client = WebAuthn::FakeClient.new(WebAuthn.configuration.allowed_origins.first)
+
+    creation_options = WebAuthn::Credential.options_for_create(
+      user: { id: @user.webauthn_id, name: @user.email_address },
+      authenticator_selection: { resident_key: "discouraged", user_verification: "discouraged" }
+    )
+    create_options = @client.create(challenge: creation_options.challenge)
+    credential = WebAuthn::Credential.from_create(create_options)
+
+    WebauthnCredential.second_factor.create!(
+      nickname: "My Security Key",
+      user: @user,
+      external_id: credential.id,
+      public_key: credential.public_key,
+      sign_count: 0
+    )
+  end
+
+  test "should return get_options" do
+    post session_path, params: { email_address: @user.email_address, password: "password" }
```

Comment:
What do you think about adding some tests for the cases where user has not initiated sign in and when has already singed in?